src/foundry/foundry.js/applications/formApplications/baseEntitySheets/permissionControl.d.ts

```diff
@@ -1,11 +1,14 @@
+import { Document } from '../../../../common/abstract/module.mjs';
```

Comment:
If you import things, stuff that should be available globally needs to be in a `declare global` block. Also, instead of importing this, just use `foundry.abstract.Document`. Also, I assume you only tried to make things compile here and did not try to type it correctly?